astropy/modeling/tests/test_models_quantities.py

```diff
@@ -809,12 +809,12 @@ def test_models_fitting(model, fitter):
     m = model["class"](**model["parameters"])
     if len(model["evaluation"][0]) == 2:
         x = np.linspace(1, 3, 100) * model["evaluation"][0][0].unit
-        y = np.exp(-x.value**2) * model["evaluation"][0][1].unit
+        y = np.exp(-(x.value**2)) * model["evaluation"][0][1].unit
         args = [x, y]
     else:
         x = np.linspace(1, 3, 100) * model["evaluation"][0][0].unit
         y = np.linspace(1, 3, 100) * model["evaluation"][0][1].unit
-        z = np.exp(-x.value**2 - y.value**2) * model["evaluation"][0][2].unit
+        z = np.exp(-(x.value**2) - y.value**2) * model["evaluation"][0][2].unit
```

Comment:
why are those parentheses added (and not around `y.value` - this just makes the expression considerably *less* clear! Does adding a space after the `-` resolve it? If not, at least write `np.exp(-(x.value**2 + y.value**2))` for now. And please raise an issue upstream - this is just inane - math precedence rules exist for a reason.

Reply:
The `-` in front of `x.value` is different from the `-` in front of `y.value`. The former is a unary minus (`__neg__()`), whereas the latter is a subtraction (`__sub__()`). In Python exponentiation has higher precedence than unary minus, so `-x.value**2` is equivalent to `-(x.value**2)`, not `(-x.value)**2`, but apparently Ruff formatter thinks it's better to be explicit.

I agree that `np.exp(-(x.value**2 + y.value**2))` would be better.

Reply:
Yes, that makes sense -- but doesn't make the ruff formatter right!